dev/interface/buzzer_interface.cpp

```diff
@@ -14,7 +14,7 @@ BuzzerIF::note_t BuzzerIF::now_playing_note = Finish;
 
 void BuzzerIF::change_playing_note(int note) {
     now_playing_note = (note_t) note;
-    if (now_playing_note > 0) {  // a valid playing_note
+    if (now_playing_note > 0) {  // A valid playing_note
```

Comment:
Actually I prefer not to capitalize for comment after code within the same line hhh. But do capitalize if the comment is the only thing in a line.

But anyway, it's no a big problem. And you can change our style guide if you want :)